config.py

```diff
@@ -64,6 +64,8 @@ class TestingConfig(BaseConfig):
     DEBUG = True
     TESTING = True
 
+    MAIL_DEFAULT_SENDER = 'certain@example.com'
```

Comment:
Does this line help? I am unsure.

Reply:
I was unsure as well, when reviewing another PR, but I think it might be needed because of the tests being made that use email feature @BadduCoder

Reply:
Yeah! that might be the case, cool then :smile: